app/presenters/link_check_csv_presenter.rb

```diff
@@ -0,0 +1,21 @@
+require 'csv'
+
+class LinkCheckCSVPresenter
+  def self.homepage_links_status_csv
+    CSV.generate do |csv|
+      csv << %w(status count)
+      LocalAuthority.group(:status).count.each do |key, value|
+        csv << [key || "nil", value]
+      end
+    end
+  end
+
+  def self.links_status_csv
+    CSV.generate do |csv|
+      csv << %w(status count)
+      Link.group(:status).count.each do |key, value|
```

Comment:
We may enable other services in future, and having the old links in the database already would give us a starting point for those. The checking takes a while, though, and would take much longer if we included all the disabled service links - we don't need quality data until we want to start using them.